releases/0.9.0-incubating/INSTALL.md

```diff
@@ -146,6 +160,7 @@ brew install gradle
 
 After running the script above, you should have all you need to install OpenWhisk. Normally there should be no error, but it
 some error messages pop-up, please [log an issue](https://github.com/apache/incubator-openwhisk/issues) for the OpenWhisk community to assist you.
+We suggest you restart your local machine to make sure all the packages have been well configured.
```

Comment:
Is there precedence for this - I don't recall we've suggested a reboot in the past.